marlowe-dashboard-client/src/Data/Map/Alignable.purs

```diff
@@ -0,0 +1,26 @@
+module Data.Map.Alignable where
+
+import Prologue
+
+import Data.Align (class Align, class Alignable)
+import Data.Map (Map)
+import Data.Map as Map
+import Data.Newtype (class Newtype)
+import Data.Set as Set
+import Data.These (maybeThese)
+
+newtype AlignableMap k v = AlignableMap (Map k v)
```

Comment:
Reading the code, I'm not sure I understand what this data type does. Would you mind adding some comments?

Reply:
Sure. It's here to add `Align` and `Alignable` instances for `Data.Map`. These instances should probably be pushed upstream to `purescript-these`.